### PR TITLE
Bug 1151925 - Bidi titles in browser chrome and card-view

### PR DESCRIPTION
**Pull request by sfoster**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=26d231bea9366353d688ae0e7c86566fdf15978c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=26d231bea9366353d688ae0e7c86566fdf15978c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f773fe529832961eb4dc9ab2d141c9e0e7571d1a

File: apps/system/style/chrome/chrome.css [hunk 1]
```diff
@@ -243,7 +248,8 @@
 
 
 /* Private browser icon in rocketbar */
-.appWindow.private > .chrome.maximized .title {
+.appWindow.private > .chrome-plain.maximized .title,
```
Comment (KevinGrandon) [inline apps/system/style/chrome/chrome.css]: Does having this not cause the padding to be double? Before this only was applied to ```.title```, and now it's also applied to ```.chrome-title-container```

Comment (sfoster) [inline apps/system/style/chrome/chrome.css]: There's no padding on .chrome-combined .title at all now, its all on .chrome-title-container. If I could make it display:inline I would, but we need block for the text-overflow ellipsis to work. For this rule, I changed the selector from chrome-plain.maximized .title to chrome-plain.maximized .title - those are exclusive - chrome-combined is never chrome-plain and vice-versa.

**Comment by autolander** [comment]
The pull request could not be integrated in as it contained a commit which was missing a bug number. Please ensure all commits contain bug numbers, and are in the format of: Bug [number] - [description]

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8e3ceb7afd6e1c0ed73d531db34d36e8c267e682

**Pull request merged**